src/components/Dashboard /StudentClassData/index.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./styles.scss";
+import moment from "moment";
+
+function StudentClassData(props) {
+  const languageMap = {
+    hi: "Hindi",
+    te: "Telugu",
+    en: "English",
+    ta: "Tamil",
+  };
+
+  return (
+    <table className="student-class-table">
+      <thead>
+        <tr>
+          <th>Class id</th>
+          <th>Date of Class </th>
+          <th>Language</th>
+          <th>Facilitator Name</th>
+          <th>Title</th>
+          <th> Feedback rating </th>
+        </tr>
+      </thead>
+
+      {props.location.state.pass.map((item) => {
```

Comment:
there should be an empty state with message 'There are no results to display' in case there is no data

Reply:
Done